From 8f71dc75b95351e9aa187aa7d5e69cf20cd42662 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Sun, 15 Jun 2025 12:51:52 +0200
Subject: [PATCH] Halium: disable DRI build

DRI support is disabled when building with Halium, so don't require it

Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>
---
Upstream-Status: Inappropriate [Halium specific]
---

diff --git a/src/media/gpu/sandbox/BUILD.gn b/src/media/gpu/sandbox/BUILD.gn
index cfcb7fa80e..7e2c67e1ed 100644
--- a/src/media/gpu/sandbox/BUILD.gn
+++ b/src/media/gpu/sandbox/BUILD.gn
@@ -33,6 +33,6 @@ source_set("sandbox") {
   if (current_cpu != "s390x" && current_cpu != "ppc64" && is_linux &&
       !is_castos) {
     # For DRI_DRIVER_DIR.
-    configs += [ "//build/config/linux/dri" ]
+    # configs += [ "//build/config/linux/dri" ]
   }
 }

